### ISSUE TITLE
Latest changes force the use of glyphicons, allow customization without forcing a glyphicon resource request.

### ISSUE THREAD

**Issue by bradplank** [plusacht/ember-bootstrap-datetimepicker#11]
The following change to ember-bootstrap-datetimepicker forces the client to make a network request for the glyphicon set when it may not be required:

```html
<span class="glyphicon glyphicon-calendar"></span>
```

I prefer the font-awesome icon set, and was using less to style the span.

**Comment by bradplank** [comment]
Could we revert the change to allow the following to work:

template:
```html
...
{{bs-datetimepicker date=date icons=faicons}}
...
```

mixin:
```javascript
import Ember from 'ember';

export default Ember.Mixin.create({
  faicons: {
    time: 'fa fa-clock-o',
    date: 'fa fa-calendar',
    up:   'fa fa-chevron-up',
    down: 'fa fa-chevron-down'
  }
});
```

controller:
```javascript
import Ember from 'ember';
import Faicons from '../../mixins/faicons';
export default Ember.ArrayController.extend(Faicons);
```

**Comment by bradplank** [comment]
Using the following in the meanwhile:

[app/templates/components/bs-datetimepicker.hbs]
```html
<div class="datetimepicker input-group date">
  {{view textFieldClass classNames="form-control" disabled=disabled name=textFieldName}}
  <span class="input-group-addon">
    <span class="fa fa-calendar"></span>
  </span>
</div>
```

**Comment by bradplank** [comment]
I have an idea to modify the component slightly, that will solve both this template update to have a default icon and allow easier customization.  I should have a pull request put together shortly.

Idea: reuse the ability to override the icons like the eonasdan-bootstrap-datetimepicker, by checking if the icons property is set, look for the calender icon setting, otherwise, use the glyphicon.

template:
```html
<span {{bind-attr class="addonIcon"}}></span>
```

**Comment by claudiocro** [comment]
This should be fixed by #13 ?

**Comment by bradplank** [comment]
+1 I agree
